examples/incremental_learning/helmet_detection/training/interface.py

```diff
@@ -26,8 +27,21 @@
 from yolo3_multiscale import Yolo3
 from yolo3_multiscale import YoloConfig
 
+
 os.environ['BACKEND_TYPE'] = 'TENSORFLOW'
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+s3_url = os.getenv("S3_ENDPOINT_URL", "http://s3.amazonaws.com")
+if not (s3_url.startswith("http://") or s3_url.startswith("https://")):
+    _url = f"https://{s3_url}"
+s3_url = urlparse(s3_url)
+s3_use_ssl = s3_url.scheme == 'https' if s3_url.scheme else True
+
+os.environ["AWS_ACCESS_KEY_ID"] = os.getenv("ACCESS_KEY_ID", "")
+os.environ["AWS_SECRET_ACCESS_KEY"] = os.getenv("SECRET_ACCESS_KEY", "")
+os.environ["S3_ENDPOINT"] = s3_url.netloc
```

Comment:
if not env ACCESS_KEY_ID, os.getenv("ACCESS_KEY_ID") will be None,
os.environ["AWS_ACCESS_KEY_ID"] = None will report error
suggest os.environ.setdefault()

Reply:
[os.getenv() method in Python returns the value of the environment variable key if it exists otherwise returns the default value.](https://docs.python.org/3/library/os.html) os.getenv(key, default = None)